apps/fz_http/lib/fz_http_web/router.ex

```diff
@@ -186,7 +192,7 @@ defmodule FzHttpWeb.Router do
     resources "/configuration", ConfigurationController, singleton: true, only: [:show, :update]
     resources "/users", UserController, except: [:new, :edit]
     resources "/devices", DeviceController, except: [:new, :edit]
-    resources "/rules", RuleController, except: [:new, :edit]
+    resources "/allow_rules", AllowRuleController, except: [:new, :edit, :update]
```

Comment:
We may want to keep the old endpoint to return an error message with a link to the upgrade process.